app/Controllers/feedController.php

```diff
@@ -318,9 +318,7 @@ public static function actualizeFeed($feed_id, $feed_url, $force, $simplePiePush
 			$feed_history = $feed->keepHistory();
 			if ($isNewFeed) {
 				$feed_history = -1; //∞
```

Comment:
Maybe it could be a good opportunity to do the same with -1 / ∞

Reply:
Ok, let's do that :)

Reply:
Done